util/autogen_banner.py

```diff
@@ -0,0 +1,49 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+from textwrap import wrap
+
+LICENSE_BANNER = [
+    "Copyright lowRISC contributors.",
+    "Licensed under the Apache License, Version 2.0, see LICENSE for details.",
+    "SPDX-License-Identifier: Apache-2.0",
+]
+
+AUTOGEN_BANNER = [
+    "-------------------------------------------------------------------------",
+    "-------- W A R N I N G: A U T O - G E N E R A T E D   C O D E !! --------",
+    "DO NOT EDIT THIS FILE DIRECTLY. GENERATED WITH THE FOLLOWING COMMAND:",
+    "-------------------------------------------------------------------------",
+]
+
+
+def get_autogen_banner(command: str, comment=None) -> str:
```

Comment:
Why not extract `command` from `sys`?

Reply:
Why is `comment` untyped? If you're going to type part of the file, type all of it.

Reply:
> Why not extract `command` from `sys`?

I will let @tjaychen & @eunchan answer this - in `topgen` the command is hardcoded in various places seemingly because some of the switches that take default values wont be present in the incantation, but it is useful to tell the reader how to change those default values if they wish to.

Reply:
Update as `comment: str`. It seems that the consuming code can use an explicit  value.

Reply:
Done.